sdk/communication/azure-communication-sms/samples/sms_sample.py

```diff
@@ -31,13 +30,30 @@ def send_sms(self):
         sms_client = SmsClient.from_connection_string(connection_string)
 
         # calling send() with sms values
-        smsresponse = sms_client.send(
-            from_phone_number=PhoneNumberIdentifier("<leased-phone-number>"),
-            to_phone_numbers=[PhoneNumberIdentifier("<to-phone-number>")],
+        sms_responses = sms_client.send(
+            from_="<leased-phone-number>",
+            to=["<to-phone-number-1>", "<to-phone-number-2>", "<to-phone-number-3>"],
             message="Hello World via SMS",
-            send_sms_options=SendSmsOptions(enable_delivery_report=True)) # optional property
-
-        print(smsresponse)
+            enable_delivery_report=True, # optional property
+            tag="custom-tag") # optional property
+        
+        failed_recipients = []
+        for sms_response in sms_responses:
+            if (sms_response.successful):
+                print("Message with message id {} was successful sent to {}"
+                .format(sms_response.message_id , sms_response.to))
+            else:
+                print("Message failed to send to {}"
+                .format(sms_response.to))
+                failed_recipients.append(sms_response.to)
+        
+        # calling send() with failed recipients
+        sms_responses = sms_client.send(
+            from_="<leased-phone-number>",
+            to=failed_recipients,
+            message="Hello World via SMS",
+            enable_delivery_report=True, # optional property
+            tag="custom-tag") # optional property
```

Comment:
I know its a bit of overkill, but maybe we should not retry for bad requests, 400?

Reply:
Sure, I can filter out the 400 requests before resending.